.github/workflows/pkgdown-netlify-preview.yaml

```diff
@@ -54,12 +54,20 @@ jobs:
           branch: gh-pages
           folder: docs
 
+      - id: deploy-dir
+        name: Determine dev status
+        run: |
+          if [[ $(grep -c -E 'sion. ([0-9]*\.){3}' ${{ github.workspace }}/DESCRIPTION) == 1 ]]; then
+            echo 'dir=./docs/dev' >> $GITHUB_OUTPUT
+          else
+            echo 'dir=./docs' >> $GITHUB_OUTPUT
+          fi
       - name: Deploy PR preview to Netlify
```

Comment:
These chnages fix broken dev previews URLs resulting from the fact we've got both release and dev versions of docs now